[quant] follow up fixes for prepare_fx/prepare_qat_fx calls in classyvision (#105)

jerryzh168 opened the pull request:
Summary:
X-link: https://github.com/fairinternal/ClassyVision/pull/105

As follow up for https://github.com/pytorch/pytorch/pull/76496, we fixes the TODOs in quantization tests
by providing correct example_inputs in the tests

Test Plan:
classyvision sandcastle and ossci

**Static Docs Preview: classyvision**
|[Full Site](https://our.intern.facebook.com/intern/staticdocs/eph/D36818665/V1/classyvision/)|

|**Modified Pages**|

Differential Revision: D36818665

vkuzo commented on torch/ao/quantization/utils.py:
```diff
@@ -495,35 +495,26 @@ def forward(self, x, key1=3, key2=3):
+            fqn_to_example_inputs[fqn] = submodule_example_inputs_tuple
+        return orig_module_call(self, *args, **kwargs)
+
+    orig_module_call = torch.nn.Module.__call__
```
should this code be in a `try` followed by L517 in a `finally`, to ensure that the call function gets patched back even if the forward encounters an exception?

jerryzh168 replied:
yeah sure

jerryzh168 committed:
[quant] follow up fixes for prepare_fx/prepare_qat_fx calls in classyvision (#793)

Summary:
X-link: https://github.com/facebookresearch/ClassyVision/pull/793

Pull Request resolved: https://github.com/pytorch/pytorch/pull/78660

X-link: https://github.com/fairinternal/ClassyVision/pull/105

As follow up for https://github.com/pytorch/pytorch/pull/76496, we fixes the TODOs in quantization tests
by providing correct example_inputs in the tests

Test Plan:
classyvision sandcastle and ossci

**Static Docs Preview: classyvision**
|[Full Site](https://our.intern.facebook.com/intern/staticdocs/eph/D36818665/V8/classyvision/)|

|**Modified Pages**|

Reviewed By: dzdang

Differential Revision: D36818665

fbshipit-source-id: ad090e4bf8820777b32540e4decfe741aeac26a7
